org.musesequencer.Muse3.json

```diff
@@ -0,0 +1,201 @@
+{
+  "app-id": "org.musesequencer.Muse3",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "muse3",
+  "rename-desktop-file": "muse.desktop",
```

Comment:
It would be nice if you could upstream those changes.

Reply:
Done muse-sequencer/muse/#636

Reply:
merged there.